src/core/model.py

```diff
@@ -4,21 +4,12 @@
 import logging
 import os.path
 import re
+from collections.abc import Generator
 from dataclasses import asdict, dataclass
 from datetime import datetime, timedelta
 from enum import Enum
 from pathlib import Path
-from typing import (
-    Any,
-    DefaultDict,
-    Dict,
-    Generator,
-    List,
-    Optional,
-    Set,
-    Tuple,
-    Union,
-)
+from typing import Any, DefaultDict, Dict, List, Optional, Set, Tuple, Union
```

Comment:
Looks like there is a few imports that is now unused in this and other files. I see we ignore `F401` in flake8 config, so it is not caught by checks. We can maybe look at that as a separate issue.

Reply:
Easy. 😅